python-checks/src/main/java/org/sonar/python/checks/UndeclaredNameUsageCheck.java

```diff
@@ -81,7 +81,7 @@ private static void checkCfgBlock(CfgBlock cfgBlock, SubscriptionContext ctx, De
         if (symbolUsage.isWrite()) {
           currentState.put(symbol, DefinedVariablesAnalysis.VariableDefinition.DEFINED);
         }
-        DefinedVariablesAnalysis.VariableDefinition varDef = currentState.getOrDefault(symbol, DefinedVariablesAnalysis.VariableDefinition.BOTTOM);
+        DefinedVariablesAnalysis.VariableDefinition varDef = currentState.getOrDefault(symbol, DefinedVariablesAnalysis.VariableDefinition.DEFINED);
```

Comment:
Why not `UNDEFINED`? Maybe it doesn't matter...

Reply:
To avoid raising an issue on global symbols. The rationale behind it is that I'll raise an issue only if I'm sure that the variable is undefined